### DISCUSSION TITLE
Adding Google Cloud Platform to a project?

### DISCUSSION THREAD

**Discussion by pierpo** [AntlerVC/firetable#1354] [Support / Q&A] [answered]
Hey,

I am trying to use the [Algolia integration](https://docs.rowy.io/field-types/connect-table) with Rowy to have connected tables.

However, it requires me to add a secret on GCP. I found [this doc](https://docs.rowy.io/how-to/add-secrets) that talks about it, but I just don't understand where my Rowy project will be linked to my GCP project?

The link on the Algolia integration page points to "https://console.cloud.google.com/run/deploy/us-central1/rowy-backend", which doesn't really make sense.
![image](https://github.com/rowyio/rowy/assets/6313316/52cbfc7b-f264-4b02-860c-6d4c6ec3b229)

(Besides, the algolia integration doc seems unfinished! There's nothing under "[Create the connection](https://docs.rowy.io/field-types/connect-table#create-the-connection)")

It might be simple but it's not very clear in the docs :)

Many thanks for you help!

**Comment by harinij** [comment, marked as answer]
Hi @pierpo, sorry for the confusion, we will update the docs soon. In the meantime. We have a field for this called [Connector](https://docs.rowy.io/field-types/connector#:~:text=Connector%20columns%20allow%20you%20to,query%20to%20third%20party%20API.) which you can use without an Algolia integration and we recommend using that.

**Reply by pierpo** [reply]
Thank you!
